ui-tests/tst/software/aws/toolkits/jetbrains/uitests/tests/CloudFormationBrowserTest.kt

```diff
@@ -116,6 +123,9 @@ class CloudFormationBrowserTest {
         waitForStackDeletion()
     }
 
+    private fun IdeaFrame.clickOn(tab: String) {
+        findAndClick("//div[@accessiblename='$tab' and @class='JLabel' and @text='$tab']")
+    }
     private fun IdeaFrame.clickOnOutputs() {
```

Comment:
nit clickOnOutputs/Events is now dead code

Reply:
yeah its actually removed in #2165

Reply:
Yeah I saw that after making this comment, looks good then